tests/conftest.py

```diff
@@ -366,18 +367,39 @@ def shutdown(self):
                                 + " and had to be killed") from None
 
 
+def kill_all_existing_windows(show_warnings=True):
+    xlsclients = subprocess.run(['xlsclients', '-l'], stdout=subprocess.PIPE)
```

Comment:
I think you can use `check_output` instead of `run` and then you do not need the `assert`

Reply:
(alternatively, passing `check=True` should also work)